core/src/test/java/org/apache/calcite/rel/rel2sql/RelToSqlConverterTest.java

```diff
@@ -2745,10 +2756,22 @@ private SqlDialect nonOrdinalDialect() {
         + "EXTRACT(MINUTE FROM TIMESTAMP '2023-12-01 00:00:00'), "
         + "EXTRACT(SECOND FROM TIMESTAMP '2023-12-01 00:00:00')\n"
         + "FROM (VALUES (0)) AS t (ZERO)";
+    final String expectedDoris = "SELECT"
+            + " EXTRACT(YEAR FROM '2023-12-01'),"
```

Comment:
The style here needs to be consistent with that of other SQL statements.

Reply:
Done